src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -228,12 +228,12 @@ public IBindingProvider BindToGenericValueProvider<TAttribute>(Func<TAttribute,
 
         // Adapter to expose a delegate veneer over IAsyncCollector builders. 
```

Comment:
Yes. It's a general pattern for making an IConverter around a lambda. 

---
In reply to: [108521945](https://github.com/Azure/azure-webjobs-sdk/pull/1093#discussion_r108521945) [](ancestors = 108521945)